lib/lm_docstache/parser.rb

```diff
@@ -128,23 +136,20 @@ def replace_variables_in_document!
             variable_replacement.call($1) :
             variable_replacement.to_s
         end
-
         text_node.content = text
       end
     end
 
     def has_skippable_variable?(text)
-      return true if hide_custom_tags.find { |pattern| text =~ pattern }
+      return true if hide_custom_tags.find { |(pattern, value)| text =~ pattern }
       !!special_variable_replacements.find do |(pattern, value)|
-        pattern = pattern.is_a?(String) ? /{{#{pattern}}}/ : /{{#{pattern.source}}}/
```

Comment:
Because we have a method that makes this happens? https://github.com/boost-legal/lm-docstache/pull/11/files#diff-473de67f52e8aca33647ef59951c5e059ee4ed9ab5fdf9287a121332db52fbbbR51